src/components/AddToShelveButton.js

```diff
@@ -1,5 +1,7 @@
 import React, {useCallback, useState} from 'react';
 import {useBooksApiService} from "../contexts/BooksApiServiceContext";
+import Button from "react-bootstrap/Button";
+import {Alert, Col, Row} from "react-bootstrap";
```

Comment:
import Button from "react-bootstrap/Button";
import {Alert, Col, Row} from "react-bootstrap";

I think you can replace with import {Button, Alert, Col, Row} from "react-bootstrap";